packages/nextjs/context/nvm/NvmContext.tsx

```diff
@@ -0,0 +1,162 @@
+      web3ProviderUri: "https://goerli-rollup.arbitrum.io/rpc",
+      web3Provider: state.provider,
+      // web3Provider: provider,
+      // The url of the marketplace api if you connect to one. It could be your own service if you run a Marketplace API
+      marketplaceUri: "https://marketplace-api.goerli.nevermined.app",
+      // The url to a Nevermined node. It could be your own if you run a Nevermined Node
+      neverminedNodeUri: "https://node.goerli.nevermined.app",
+      // The public address of the above Node
+      neverminedNodeAddress: "0x5838B5512cF9f12FE9f2beccB20eb47211F9B0bc",
+      // The url to access the nevermined subgraphs required to query for on-chain events
+      graphHttpUri: "https://api.thegraph.com/subgraphs/name/nevermined-io/public",
+      // Folder where are copied the ABIs of the Nevermined Smart Contracts
+      artifactsFolder: "http://localhost:3000/artifacts",
+      marketplaceAuthToken:
+        localStorage.getItem("marketplaceAuthToken") != null
+          ? (localStorage.getItem("marketplaceAuthToken") as string)
+          : "",
+    };
+    setNvmConfig(config);
+    try {
+      const sdk: Nevermined = await Nevermined.getInstance(config);
+      setNvm(sdk);
+      console.log(await sdk.utils.versions.get());
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const login = async () => {
+    try {
+      if (!nevermined) return;
+
+      const publisher = nevermined.accounts.getAccount(address as string);
+
+      const clientAssertion = await nevermined.utils.jwt.generateClientAssertion(publisher);
+      const loginResult = await nevermined.services.marketplace.login(clientAssertion);
+      localStorage.setItem("marketplaceAuthToken", loginResult);
+      const payload = decodeJwt(loginResult);
+      debugger
+      setState(prevState => ({ ...prevState, payload, publisher }));
+    } catch (error) {
+      console.error;
+    }
+  };
+
+  /**
+   * 
+   * @info That is currently pushing a mock asset
+   * @todo Publish asset inserted by user (pass in all the props)
+   */
+  const publishAsset = async () => {
+    try {
+      debugger
+      const { publisher, payload } = state;
+      const assetPrice = new AssetPrice(publisher.getId(), 0n);
+
+      const metadata = getMetadata();
+      metadata.main.name = `${metadata.main.name} - ${Math.random()}`;
+      metadata.userId = payload?.sub;
+
+      const assetAttributes = AssetAttributes.getInstance({
+        metadata,
+        services: [
+          {
+            serviceType: "access",
+            price: assetPrice,
+          },
+        ],
+        providers: config?.neverminedNodeAddress ? [config?.neverminedNodeAddress] : undefined,
+      });
+      const steps = [];
+      const ddo = await nevermined?.assets.create(assetAttributes, publisher).next(step => steps.push(step));
+      console.log("ddo", ddo);
+    } catch (error) {
+      console.error;
```

Comment:
fix: actually log the error

```suggestion
      console.error(error);
```